### ISSUE TITLE
Consider to make the main inserter scrollable div focusable across all browsers

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#7576]
Something that came into my mind while testing #6261 

In Firefox, scrollable elements (for example because they have a fixed height, a long content, and they use overflow: auto) become focusable. This is a historical Firefox feature and it's intended to give keyboard users the ability to scroll the div using the arrow keys.

See below (outlined in red) the scrollable div while using the arrow keys to scroll it:

![screen shot 2018-06-27 at 18 42 07](https://user-images.githubusercontent.com/1682452/41987639-09e657ec-7a3a-11e8-987a-322bea206db8.png)

In other browsers (Chrome, Safari, Edge) scrollable divs are not focusable. This is a problem for accessibility. The best option would be making the scrollable div always focusable across all browsers. For more details see this post by Mr. Steve Faulkner: https://developer.paciellogroup.com/blog/2016/02/short-note-on-improving-usability-of-scrollable-regions/

Worth noting there are also bugs open against [Chrome](https://code.google.com/p/chromium/issues/detail?id=584618) and [Edge](https://connect.microsoft.com/IE/feedbackdetail/view/2324891/css-overflow-auto-does-not-allow-scroll-via-keyboard).

The technique is simple and just requires to add a tabindex 0, an ARIA role and aria-label, e.g.:
`tabindex="0" role="region" aria-label="Some meaningful text here"`

I seem to recall the inserter results div used to have a tabindex=0 at some point, probably something that got lost during the recent refactoring. Regardless, I'd recommend to make the user experience consistent across all browsers.

**Comment by afercia** [comment]
So in Gutenberg 2.4 the inserter "panel" with the block types was focusable and properly labeled. The focus style was very simple:

<img width="473" alt="screen shot 2018-07-02 at 14 24 36" src="https://user-images.githubusercontent.com/1682452/42163787-27e0bd00-7e04-11e8-947c-3d1e2fd1fa00.png">

I'd recommend to make it focusable again, to pair the behavior across different browsers.
